app/assets/javascripts/darkswarm/all.js.coffee

```diff
@@ -59,3 +60,4 @@ $ ->
   # Hacky fix for issue - http://foundation.zurb.com/forum/posts/2112-foundation-5100-syntax-error-in-js
   Foundation.set_namespace ""
   $(document).foundation()
+  
```

Comment:
Your editor (or the original author's editor) seems to do something weird with file endings.